lib/src/cupertino/cupertino_toolbar.dart

```diff
@@ -129,19 +143,25 @@ class _CupertinoPopoverToolbarState extends State<CupertinoPopoverToolbar> {
       focalPoint: widget.focalPoint,
       allowHorizontalArrow: false,
       padding: widget.padding,
+      useArrowArea: true,
       child: _buildContent(),
     );
   }
 
   Widget _buildContent() {
+    // Expand the height so the buttons can be displayed over the arrow.
+    // Multiply by two because the menu reserves space for the arrow both
+    // above and bellow the content.
+    final height = widget.height + (widget.arrowLength * 2);
```

Comment:
This seems like the wrong place for this behavior. First, widget building shouldn't know anything about its own render object behavior, but it definitely shouldn't know about the render object behavior of a different widget.

If the popover menu is the one that's positioning content over an arrow, then shouldn't the popover menu be the one imposing the final height?

Reply:
With the lastest commit, the popover menu enforces the final height.